libs/island-ui/contentful/src/lib/AssetLink/AssetLink.tsx

```diff
@@ -11,14 +11,19 @@ export const AssetLink: FC<React.PropsWithChildren<AssetLinkProps>> = ({
   url,
   children,
 }) => {
+  let extension: string
```

Comment:
I myself am not a huge fan of uninitialised variables and would prefer it if in line 16 you would define the variable inline.